packages/db/src/models/granule.ts

```diff
@@ -82,6 +82,25 @@ export default class GranulePgModel extends BasePgModel<PostgresGranule, Postgre
     return super.get(knexOrTransaction, params);
   }
 
+  /**
+   * FUTURE: This should not be used to fetch unique granules. Granules
+   * can should be queried by cumulus_id or a combination of granule_id +
+   * collection_cumulus_id. This function is provided for legacy support.
+   *
+   * Fetches a single granule from PostgreSQL by a granuleId
+   *
+   * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+   * @param {string} granuleId - A Granule's granule_id
+   * @returns {Promise<PostgresGranuleRecord>} The returned record
+   */
+  getByGranuleId(
+    knexOrTransaction: Knex | Knex.Transaction,
+    granuleId: string
+  ): Promise<PostgresGranuleRecord> {
+    // TODO use getUniqueGranuleByGranuleId?
```

Comment:
Address TODO?

Reply:
Yeah, I removed this function altogether in favor of `getUniqueGranuleByGranuleId:

https://github.com/nasa/cumulus/pull/2415/commits/e1e5bdfce103ab62e3f1bbc9ecede42c0f7ea1be